sessionmanager.cpp

```diff
@@ -105,10 +105,16 @@ void SessionManager::openSession(const QHash<QString, QString>& port_cfg)
         serial->close();
 
     // configure port
-    //
+#ifdef Q_OS_MAC
```

Comment:
we will probably be able to get rid of that when upgrading to QT > 5.5, as the [QSerialPort  OSX bug seems to be corrected](https://bugreports.qt.io/browse/QTBUG-45127)